doc/bibliography.rst

```diff
@@ -170,3 +170,15 @@ Bibliography
    | J. F. Ziegler, J. P. Biersack, and U. Littmark
    | In *The Stopping and Range of Ions in Matter*, volume 1
    | New York, 1985. Pergamon. ISBN 0-08-022053-3
+
```

Comment:
Check if the references here are in alphabetic order. If so, follow the convention.